### PR TITLE
Example component creation

### PR DESCRIPTION
**Pull request by andyhqtran**
**Do not merge**
Thisi s an example PR

### CODE REVIEW

File: packages/zephyr/stories/Modal.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Modal, ModalProps } from '../src/Modal';
+
+const meta = {
+  title: 'Zephyr/Modal',
```
Comment (kylemh) [inline packages/zephyr/stories/Modal.stories.tsx:5]: I think we don't need to do `Zephyr/` and can make sure it's globally applied

Comment (andyhqtran) [reply packages/zephyr/stories/Modal.stories.tsx:5]: Would we be able to define the categories per packages? Definitely would be a huge time saver if we could.

Comment (kylemh) [reply packages/zephyr/stories/Modal.stories.tsx:5]: Maybe something like this in the very long-term, but a cursory glance didnt reveal anything we could do on a package basis - only globally. So, carry on!

Comment (andyhqtran) [reply packages/zephyr/stories/Modal.stories.tsx:5]: That'd be a great thing to do! Maybe storybook will release something like this by then 👀 

(Also this PR is just a reference for @lin-jessica and won't be merged in)

**Comment by dwilt** [comment]
@andyhqtran what is the scoop with this PR?

**Comment by andyhqtran** [comment]
> @andyhqtran what is the scoop with this PR?

Closing this now, this was just a example PR to get @lin-jessica up to speed a while back.

**Pull request closed without merging**